src/stravalib/client.py

```diff
@@ -559,7 +561,7 @@ def get_club_members(
 
     def get_club_activities(
         self, club_id: int, limit: int | None = None
-    ) -> BatchedResultsIterator[model.Activity]:
+    ) -> BatchedResultsIterator[model.ClubActivity]:
```

Comment:
This is a new(ish) to the spec class that we hadn't brought into our model module yet.